src/renderer/index.css

```diff
@@ -78,6 +78,24 @@
     background: rgba(0, 0, 0, 0.25);
 }
 
+.tippy-box[data-theme~='default'] {
+    @apply !bg-neutral-900;
+    @apply !text-white;
```

Comment:
I beleave you don't need the `!` to make the class `!important`
It was only when we wanted to try with no theme 
(same for others)